SmartDeviceLinkTests/RPCSpecs/StructSpecs/SDLKeyboardPropertiesSpec.m

```diff
@@ -33,28 +33,41 @@
         testStruct.keypressMode = testMode;
```

Comment:
One last thing, the last two tests should check if `autoCompleteText` has been set to `nil`.

```
it(@"Should get correctly when initialized with initWithLanguage:keyboardLayout:keypressMode:limitedCharacterList:autoCompleteList:", ^ {
    SDLKeyboardProperties *testStruct = [[SDLKeyboardProperties alloc] initWithLanguage:testLanguage keyboardLayout:testLayout keypressMode:testMode limitedCharacterList:testLimitedCharacterList autoCompleteList:testAutoCompleteList];
    expect(testStruct.language).to(equal(testLanguage));
    expect(testStruct.keyboardLayout).to(equal(testLayout));
    expect(testStruct.keypressMode).to(equal(testMode));
    expect(testStruct.limitedCharacterList).to(equal(testLimitedCharacterList));
    expect(testStruct.autoCompleteList).to(equal(testAutoCompleteList));
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wdeprecated-declarations"
    expect(testStruct.autoCompleteText).to(beNil());
#pragma clang diagnostic pop
});

it(@"Should return nil if not set", ^ {
    SDLKeyboardProperties* testStruct = [[SDLKeyboardProperties alloc] init];
    
    expect(testStruct.language).to(beNil());
    expect(testStruct.keyboardLayout).to(beNil());
    expect(testStruct.keypressMode).to(beNil());
    expect(testStruct.limitedCharacterList).to(beNil());
    expect(testStruct.autoCompleteList).to(beNil());
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wdeprecated-declarations"
    expect(testStruct.autoCompleteText).to(beNil());
#pragma clang diagnostic pop
});
```